pages/common/piper.md

```diff
@@ -0,0 +1,28 @@
+# piper
+
+> A fast, local neural text to speech system that sounds great.
+> More information: <https://github.com/rhasspy/piper>.
+
+- Output a wav [f]ile using a [m]odel (assuming config json at path to model_path + .json):
+
+`echo {{Thing to say}} | piper -m {{path/to/model.onnx}} -f {{outputfile.wav}}`
+
+- Output a wav [f]ile using a [m]odel and specifying its [c]onfig json:
+
+`echo {{Thing to say}} | piper -m {{path/to/model.onnx}} -c {{path/to/model.onnx.json}} -f {{outputfile.wav}}`
+
+- Select a particular speaker in a voice with multiple speakers:
+
+`echo {{'Warum?'}} | piper -m {{de_DE-thorsten_emotional-medium.onnx}} --speaker {{1}} -f {{angry.wav}}`
+
+- Stream output immediately to mpv (synthesizing second sentence while first sentence is spoken):
+
+`echo {{'First sentence. Second sentence'}} | piper -m {{en_GB-northern_english_male-medium.onnx}} --output-raw -f - | mpv -`
+
+- Speak twice as fast, with huge gaps between sentences:
+
+`echo {{'Speaking twice the speed. With added drama!'}} | piper -m {{foo.onnx}} --length_scale {{0.5}} --sentence_silence {{2}} -f {{drama.wav}}`
+
+- Download model and config after trying it out on <https://rhasspy.github.io/piper-samples/>:
+
+`export MODEL="{{en_GB-alba-medium}}"; for EXT in .onnx .onnx.json; do curl -L "https://huggingface.co/rhasspy/piper-voices/resolve/main/{{en/en_GB/alba/medium/}}$MODEL$EXT" > $MODEL$EXT; done`
```

Comment:
Not sure if I'd keep this example ... After all, this demonstrates how to download files, and not the usage of piper

Reply:
I was torn on this but included it because:
1. there's only one place where you will download models, and it would be super useful to see that in one of the examples
2. I noticed that `ollama` includes an example for `curl`ing an update, which seemed to me a similar level of relatedness (arguably less so)
3. piper-tts is literally useless without at least one model, and I don't know how to document it succinctly such that someone would know how to rectify this without giving up on the `tldr` page and looking at the github repo

Of course, I can remove it, but it'd be nice to address no. 3 somehow in its place

Reply:
> Is it perhaps ok to include a link to https://rhasspy.github.io/piper-samples/ somewhere in the description of the project? As far as I can see though that doesn't seem to be done anywhere else, with the only link being to the project page itself.

Yep, you can include an additional line of page description, if it's really necessary. We have a few pages like [pacman](https://github.com/tldr-pages/tldr/blob/main/pages/linux/pacman.md?plain=1) doing the same.

Reply:
Great, well I've done that now, and removed the `curl` example. Does that work for everyone?